src/spatial/views/satellite_predictions.py

```
@@ -0,0 +1,45 @@
+from datetime import datetime, timezone
+
+import gcsfs
```

Ruff

`gcsfs` imported but unused

Remove unused import: `gcsfs`

(F401)